carbon_black_cloud/help.md

```diff
@@ -0,0 +1,269 @@
+# Description
+
+The VMware Carbon Black Cloud is a cloud-native endpoint protection platform (EPP) that combines the intelligent system hardening and behavioral prevention needed to keep emerging threats at bay, using a single lightweight agent and an easy-to-use console
+
+# Key Features
+
+* Get device information
+
+# Requirements
+
+* API Key
```

Comment:
Could summarize these to one line called API credentials. Since we list them out in the Setup section